app/src/main/java/com/kickstarter/viewmodels/PledgeFragmentViewModel.kt

```diff
@@ -1391,6 +1382,18 @@ interface PledgeFragmentViewModel {
                 ) { s, t, b -> checkoutData(s, t, NumberUtils.parse(b), null) }
                     .compose<Pair<CheckoutData, PledgeData>>(combineLatestPair(pledgeData))
 
+            this.onRiskManagementConfirmed
+                .compose<Pair<Void, PledgeReason>>(combineLatestPair(pledgeReason))
+                .filter {
+                    it.second == PledgeReason.PLEDGE
+                }
+                .compose(combineLatestPair(checkoutAndPledgeData))
+                .filter { it.second.second.pledgeFlowContext() == PledgeFlowContext.NEW_PLEDGE }
+                .subscribe {
+                    this.analyticEvents.trackPledgeConfirmCTA(it.second.first, it.second.second)
```

Comment:
Could you add on the PledgeFragmentViewModelTest https://github.com/kickstarter/android-oss/blob/f6d5288901a94ebdfe278e30a07acc646bc5881b/app/src/test/java/com/kickstarter/viewmodels/PledgeFragmentViewModelTest.kt#L2117

the assert for the analytics as well? 🙏